demo/src/index.tsx

```diff
@@ -3,10 +3,14 @@ import { createRoot } from "react-dom/client";
 import App from "./App";
 import "@picocss/pico";
 import "./css/styles";
+("use client");
+import { ErrorBoundary } from "react-error-boundary";
 
 const root = createRoot(document.getElementById("app"));
 root.render(
     <StrictMode>
-        <App />
+        <ErrorBoundary fallback={<div>Something went wrong</div>}>
```

Comment:
So I was expecting this to wrap just the output (`SvgDisplay`), did you not get that to work?

Reply:
Fixed, I moved the `Output` heading out of `SvgDisplay`

![image](https://github.com/david-yz-liu/memory-models-rough/assets/67441706/14c3fbd8-fa01-4804-a68a-0e9bd82f99e8)